frontend/public/components/deployment-config.tsx

```diff
@@ -97,6 +118,10 @@ export const DeploymentConfigsDetails: React.SFC<{obj: any}> = ({obj: dc}) => {
   return <React.Fragment>
     <div className="co-m-pane__body">
       <SectionHeading text="Deployment Config Overview" />
+      {dc.spec.paused && <p className="alert alert-info">
+        <span className="pficon pficon-info" aria-hidden="true"></span>
+        <b>{dc.metadata.name} is paused.</b> This will stop any new rollouts or triggers from running until resumed. <a className="clickable" onClick={() => pause(dc).catch(pauseError)}>Resume Rollouts</a>
```

Comment:
```suggestion
        <b>{dc.metadata.name} is paused.</b> This will stop any new rollouts or triggers from running until resumed. <button className="btn btn-link" type="button" onClick={() => pause(dc).catch(pauseError)}>Resume Rollouts</button>
```